src/scenes/AcceleratorRouter/Documents/DocumentView/DocumentOutlinePanel.tsx

```diff
@@ -0,0 +1,239 @@
+import React, { useCallback, useEffect, useRef, useState } from 'react';
+
+import {
+  Box,
+  ButtonBase,
+  IconButton,
+  Tooltip,
+  TooltipProps,
+  Typography,
+  styled,
+  tooltipClasses,
+  useTheme,
+} from '@mui/material';
+
+import Icon from 'src/components/common/icon/Icon';
+import { useIsVisible } from 'src/hooks/useIsVisible';
+import { requestListVariablesValues } from 'src/redux/features/documentProducer/values/valuesThunks';
+import { selectVariablesWithValues } from 'src/redux/features/documentProducer/variables/variablesSelector';
+import { requestListVariables } from 'src/redux/features/documentProducer/variables/variablesThunks';
+import { useSelectorProcessor } from 'src/redux/hooks/useSelectorProcessor';
+import { useAppDispatch, useAppSelector } from 'src/redux/store';
+import { Document as DocumentType } from 'src/types/documentProducer/Document';
+import { SectionVariableWithValues, VariableWithValues } from 'src/types/documentProducer/Variable';
+
+const isHTMLElementInViewport = (element: HTMLElement): boolean => {
+  const rect = element.getBoundingClientRect();
+
+  return (
+    rect.top >= 0 &&
+    rect.left >= 0 &&
+    rect.bottom <= (window.innerHeight || document.documentElement.clientHeight) &&
+    rect.right <= (window.innerWidth || document.documentElement.clientWidth)
+  );
+};
+
+const StyledTooltip = styled(({ className, ...props }: TooltipProps) => (
```

Comment:
I think the tooltip component from web-components already has this colors

Reply:
I thought these changes felt familiar, will update. Thanks! 🙌

Reply:
I tried using `Tooltip` from `web-components`, but it broke the layout for some reason. Will spend a few minutes trying to fix the layout issues, but will revert if I'm unable to work it out relatively quick.

## Before

![localhost_3000_accelerator_documents_1_tab=document (1)](https://github.com/user-attachments/assets/adce9b18-f245-4a55-aa0d-fd0f6a66ddf9)

## After

![localhost_3000_accelerator_documents_1_tab=document](https://github.com/user-attachments/assets/e4097cd7-78fa-4e76-961e-417abaa04897)

Reply:
Oh, that's strange. Yeah, don't worry to much if it takes time. I thought it was going to be an easy change